recipe/meta.yaml

```diff
@@ -52,6 +52,9 @@ requirements:
     - redis-py >=3.0.0
     #django
     - django >=3.2
+    # zoneinfo
+    #- tzdata >=2024.1  # [win]
```

Comment:
`tzdata` is still an extra dependency for windows, see https://github.com/HypothesisWorks/hypothesis/blob/5578efc734aeeaf3375b69e2d8fc77de9d80cc9a/hypothesis-python/setup.py#L67